src/personalia.utils.ts

```diff
@@ -0,0 +1,36 @@
+import { IPersonalia } from "./types/personalia.types";
+
+export function getAge(date: string) {
```

Comment:
Hvis dette er hjelpefunksjoner bare for personalia så tenker jeg vi kan legge filen i samme mappe som komponenten eller bare inline hvis komponenten er liten

Reply:
Enig, flytter den. :)